scss/_patterns_card.scss

```diff
@@ -31,15 +9,28 @@
     overflow: auto; // prevent overflow of children margins
     padding: $spv-intra--scaleable;
 
-    .p-card__content {
-      margin-top: - $sp-unit * 2;
+    img {
+      margin-bottom: $spv-inter--condensed-scaleable;
+      vertical-align: top;
     }
   }
 
+  %highlight {
+    box-shadow: 0 1px 5px 1px transparentize($color-dark, .8);
+  }
+
+  p.p-card__content { // sass-lint:disable-line no-qualifying-elements - this is because the class is applied to both divs and ps. ps have special rules to ring them closer to other text elements.
```

Comment:
This is over selected. Please don't use elements when selecting.

Reply:
fixed